spec/jasmine-test-runner.coffee

```diff
@@ -10,6 +10,16 @@ module.exports = ({logFile, headless, testPaths, buildAtomEnvironment}) ->
   window[key] = value for key, value of require '../vendor/jasmine'
   require 'jasmine-tagged'
 
+  # Rewrite global jasmine functions to have support for async tests.
+  # This way packages can create async specs without having to import these from the
+  # async-spec-helpers file.
+  global.it = asyncifyJasmineFn global.it
+  global.fit = asyncifyJasmineFn global.fit, 1
```

Comment:
Can you help me understand why we need to specify the `posCallback` arg for `fit`, but not for `ffit` and `fffit`?

Reply:
`fit` is the only of these methods that does not have the callback as the last argument ([original source code](https://github.com/atom/jasmine-focused/blob/master/src/jasmine-focused.coffee#L30)), so I need to override the position of the callback.

Now that I look at it, this may be confusing for the future reader, so I'm gonna update the PR to make the position of the callback required and always pass it.